01-getting-started/projects/final/app/src/main/java/com/raywenderlich/podplay/util/CPUStress.kt

```diff
@@ -0,0 +1,41 @@
+package com.raywenderlich.podplay.util
```

Comment:
Copyright is missing

Reply:
Good finding! Will add copyright.

Reply:
Assigned @zr-faisal - https://trello.com/c/45BTAx2e/27-ensure-copyright-heading-is-added-to-each-class